guides/user/canary/judge/index.md

```diff
@@ -66,10 +66,14 @@ This is the step that compares the canary and baseline data for each included
 metric. The output of this step is a classification, for each metric indicating
 if there is a significant difference between the canary and baseline.
 
-Each metric is classified as either “Pass”, “High”, or “Low” as shown in the
+Each metric is classified as either “Pass”, “High”, or “Low,” as shown in the
```

Comment:
I'd make the comma placement consistent with the other members of the list.

Reply:
Yeah, so would I.
Thanks ;)